readme.md

```diff
@@ -427,6 +427,34 @@ If the field is itself an object containing more fields, its _type_ should be "n
 }
 ```
 
+### Default Value Support for our components
+This is to support default values for our components. In each config, we can represent a defaultValue key. Custom Function will be called, when the
+component does not have value.
+
+```
+{
+    "title": "Text",
+    "editable": true,
+    "display": true,
+    "type": "text",
+    "defaultValue": () => "Initial Default Value"
```

Comment:
call it defaultValueFn or defaultValueEvaluator, latter is better

Reply:
updated to defaultValueFn